makefile

```diff
@@ -49,6 +49,8 @@ package:: create-package-folder package-linux package-windows
 
 release:: clean checkstyle release-test pre-release build prepack package
 
+release-arm:: clean checkstyle release-test pre-release build-arm prepack-arm package-arm
```

Comment:
Can making the following changes avoid create another release rule called release-arm? 

Move build-arm to build:: 
Move prepack-arm to prepack:: 
Move package-arm to package-linux: